app/views/past_events/index.html.erb

```diff
@@ -0,0 +1,126 @@
+<div class="timeline">
+	<ul>
+		<li>
+			<div class="content">
+				<h3>Tasasek: The Courage To Choose</h3>
+				<img src="../assets/2018logo.jpg" class="logo" alt="Tasasek">
```

Comment:
Use `image_tag`

Reply:
Yes, I have updated it on the other branch, will be merged to development branch soon.